recipe/meta.yaml

```diff
@@ -35,7 +35,7 @@ requirements:
     - libxml2
     - openjpeg
     - pango
-    - perl
+    - perl *
```

Comment:
I'm not sure this is the best approach here. I'm not "in" on the perl migration details. Maybe @isuruf or @mbargull can shed some light on the best approach here.

Reply:
Actually @mbargull suggested this approach but I‘m open to discuss that.

Reply:
If Marcel approves I don't have anything else to add :smile:

Reply:
:hugs: